server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const port = 3000;
```

Comment:
Consider using environment variables for sensitive data like the port number. This will make your application more secure and configurable. [important]